src/modules/flight_mode_manager/tasks/AutoFollowTarget/follow_target_params.c

```diff
@@ -95,7 +95,7 @@ PARAM_DEFINE_FLOAT(FLW_TGT_FA, 180.0f);
  *
  * Maintain altitude or track target's altitude. When maintaining the altitude,
  * the drone can crash into terrain when the target moves uphill. When tracking
- * the target's altitude, the follow altitude NAV_MIN_FT_HT should be high enough
+ * the target's altitude, the follow altitude FLW_TGT_HT should be high enough
```

Comment:
@julianoes The only change I made was this one, through the github interface. Then all these other updates in other files appeared. Have you any idea what might have caused this? Do I need to be concerned?